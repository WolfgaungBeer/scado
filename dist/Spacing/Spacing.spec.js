'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

var _Wrapper = require('../../__mocks__/Wrapper');

var _Wrapper2 = _interopRequireDefault(_Wrapper);

var _ = require('./');

var _2 = _interopRequireDefault(_);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

it('renders the Spacing Component correctly', function () {
    var tree1 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default,
            null,
            'Spacing'
        )
    )).toJSON();
    expect(tree1).toMatchSnapshot();

    var tree2 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default,
            { scale: 'xxl', top: true },
            'Spacing'
        )
    )).toJSON();
    expect(tree2).toMatchSnapshot();

    var tree3 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default,
            { scale: 'xxl', left: true },
            'Spacing'
        )
    )).toJSON();
    expect(tree3).toMatchSnapshot();

    var tree4 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default,
            { scale: 'xxl', bottom: true },
            'Spacing'
        )
    )).toJSON();
    expect(tree4).toMatchSnapshot();

    var tree5 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(
            _2.default,
            { scale: 'xxl', right: true },
            'Spacing'
        )
    )).toJSON();
    expect(tree5).toMatchSnapshot();
});