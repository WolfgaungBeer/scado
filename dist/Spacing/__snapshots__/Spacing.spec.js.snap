// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Spacing Component correctly 1`] = `
<div
  className="sc-jKJlTe gQiYVm"
  scale={undefined}
/>
`;

exports[`renders the Spacing Component correctly 2`] = `
<div
  className="sc-jKJlTe bdWLxj"
  scale="xxl"
/>
`;

exports[`renders the Spacing Component correctly 3`] = `
<div
  className="sc-eNQAEJ jrmYtL"
  scale="xxl"
>
  Spacing
</div>
`;

exports[`renders the Spacing Component correctly 4`] = `
<div
  className="sc-eNQAEJ ciqfQ"
  scale="xxl"
>
  Spacing
</div>
`;

exports[`renders the Spacing Component correctly 5`] = `
<div
  className="sc-eNQAEJ BJbUw"
  scale="xxl"
>
  Spacing
</div>
`;
